$OpenBSD$

Dispense with the need to use gnu grep (incompatible command line options).

Index: CM_linux_packages.cmake
--- CM_linux_packages.cmake.orig
+++ CM_linux_packages.cmake
@@ -1,6 +1,6 @@
 # determine the version number from the #define in libyuv/version.h
 EXECUTE_PROCESS (
-	COMMAND grep --perl-regex --only-matching "(?<=LIBYUV_VERSION )[0-9]+" include/libyuv/version.h
+	COMMAND awk "/^#define LIBYUV_VERSION/ {print \$3}" include/libyuv/version.h
 	WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
 	OUTPUT_VARIABLE YUV_VERSION_NUMBER
 	OUTPUT_STRIP_TRAILING_WHITESPACE )
